internal/commands/pull/command.go

```diff
@@ -108,6 +131,12 @@ func (cmd *Command) Inputs() cli.InputResolver {
 
 // Handler is the command handler
 func (cmd *Command) Handler(profile *user.Profile, ui terminal.UI, clients cli.Clients) error {
+	if cmd.inputs.IncludeNodeModules && cmd.inputs.IncludePackageJSON {
```

Comment:
this type of validation is logic that should live in `inputs.Resolve`

the distinction b/w this is that, when an error happens in `Handler` we never print the "usage info" about the command, we just simply print the error.

when an error occurs in `Resolve`, we'll still print the "usage info"

so in a way, `Resolve` should return errors that are related to flags/inputs and those types of invariants.  but once we know input values are valid, most other errors should arise from `Handler` (usually will be an API/network error)